app/javascript/packages/document-capture/components/document-capture.jsx

```diff
@@ -9,14 +9,46 @@ import MobileIntroStep from './mobile-intro-step';
 import DeviceContext from '../context/device';
 import ServiceProviderContext from '../context/service-provider';
 import Submission from './submission';
+import SubmissionStatus from './submission-status';
 import DesktopDocumentDisclosure from './desktop-document-disclosure';
 import useI18n from '../hooks/use-i18n';
+import { RetrySubmissionError } from './submission-complete';
+import SuspenseErrorBoundary from './suspense-error-boundary';
+import SubmissionInterstitial from './submission-interstitial';
 
 /** @typedef {import('react').ReactNode} ReactNode */
 /** @typedef {import('./form-steps').FormStep} FormStep */
 /** @typedef {import('../context/upload').UploadFieldError} UploadFieldError */
 
-function DocumentCapture() {
+/**
+ * Returns a new object with specified keys removed.
+ *
+ * @template {Record<string,any>} T
+ *
+ * @param {T} object Original object.
+ * @param {...string} keys Keys to remove.
+ *
+ * @return {Partial<T>} Object with keys removed.
+ */
+export const omit = (object, ...keys) =>
```

Comment:
> Not that our Ruby and JS _need_ to match, but WDYT of renaming this `except`, to be similar to the ActiveSupport extension? https://api.rubyonrails.org/classes/Hash.html#method-i-except

No strong preference, but I'd named it as matching the equivalent function from Lodash:

https://lodash.com/docs/4.17.15#omit

There may be a stronger case in favor of your suggestion though, since we actually use Rails, and not Lodash 😄

Reply:
> Not that our Ruby and JS _need_ to match, but WDYT of renaming this `except`, to be similar to the ActiveSupport extension? https://api.rubyonrails.org/classes/Hash.html#method-i-except

Renamed in a1d9084.